test_suite/tests/json.rs

```diff
@@ -54,30 +44,27 @@ fn test_unit_struct() {
 	struct UnitStruct;
 
 	assert_json_for_type::<UnitStruct>(json!({
-		"id": {
-			"custom.name": 1,
-			"custom.namespace": [2],
-			"custom.params": [],
-		},
-		"def": {
-			"tuple_struct.types": []
+		"composite": {
+			"name": 1,
+			"namespace": [2],
```

Comment:
Could we maybe serialize `name` and `namespace` as combined `path` instead? So a `path` is basically a non-empty sequence of `name-IDs`, e.g. `[ 42 ]` stands only for the name represented by 42 and `[ 5, 2, 42 ]` stands for the path `foo::bar::Baz` if `foo`, `bar` and `Baz` are represented by IDs 5, 2 and 42 respectively. Please tell me what you do think about this idea.

Reply:
On the Rust side we should add getters for `name` and `namespace` to `Path` if they don't already exist.